api/agent/drivers/docker/docker.go

```diff
@@ -615,27 +608,13 @@ func cherryPick(ds *docker.Stats) drivers.Stat {
 	}
 }
 
-func (drv *DockerDriver) startTask(ctx context.Context, container string) error {
-	log := common.Logger(ctx)
-	log.WithFields(logrus.Fields{"container": container}).Debug("Starting container execution")
-	err := drv.docker.StartContainerWithContext(container, nil, ctx)
-	if err != nil {
-		dockerErr, ok := err.(*docker.Error)
```

Comment:
Extra complication due to retries, based on https://github.com/iron-io/runner/commit/22a867ac0a575880d15bd1ea3768decabe417652